munit/jvm/src/main/scala/munit/internal/package.scala

```diff
@@ -0,0 +1,13 @@
+package munit
+
+package object internal {
+  type File = java.io.File
```

Comment:
Can we add an object called `PlatformIO` that has these methods and aliases instead? I try to avoid package objects when possible.

Reply:
I moved the io-related ones to `PlatformIO`. I moved the reflect ones to `PlatformCompat`.